park-dashboard/src/components/NationalParksMap/TooltipComponent.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import MapChart from "./MapChart";
+import ReactTooltip from "react-tooltip";
+import { Card } from "react-bootstrap";
+
+const TooltipComponent = ({ stateWiseNationalParks }) => {
+  const [name, setName] = useState("");
+  const [address, setAddress] = useState("");
+  const [city, setCity] = useState("");
+  const [postCode, setPostCode] = useState("");
+  const [description, setDescription] = useState("");
+  const [contact, setContact] = useState("");
+
+  useEffect(() => {
+    ReactTooltip.rebuild();
+  }, []);
+  return (
+    <div>
+      <MapChart
+        stateWiseNationalParks={stateWiseNationalParks}
+        setTooltipName={setName}
+        setTooltipAddress={setAddress}
+        setTooltipCity={setCity}
+        setTooltipPostCode={setPostCode}
+        setTooltipDescription={setDescription}
+        setTooltipContact={setContact}
+      ></MapChart>
+
+      {name && (
+        <ReactTooltip
```

Comment:
The card body looks pretty nice. You did a nice job by adding the card component here. It looks really nice.